src/components/ui/input/URLInput.tsx

```
@@ -0,0 +1,332 @@
+
+  // Generate unique IDs for accessibility
+  const inputId = useUniqueId('url-input');
+  const descriptionId = useUniqueId('url-description');
+  const errorId = useUniqueId('url-error');
+  const validationErrorId = useUniqueId('url-validation-error');
+  
+  // Create aria-describedby string
+  const ariaDescribedBy = [
+    displayDescription ? descriptionId : null,
+    displayError ? errorId : null,
+    showValidation && value && !isURLValid && !displayError ? validationErrorId : null
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)